docs/release_notes/release_notes.md

```diff
@@ -9,46 +9,63 @@ has_children: true
+        * AWS - Concurrent Sessions From Different IPs
+        * AWS - Multiple Failed MFA Requests For User
+        * AWS - Created a Policy Version that allows All Resources to be Accessed
+        * AWS - Someone Tries to Retrieve the Encrypted Administrator Password
+        * AWS - RDS Master User Password has been Reset
+        * AWS - Bucket Versioning is Disabled
+        * AWS - Multi Factor Authentication is Disabled for IAM User
+        * AWS - Successful Login From Unusual Country
+        * AWS - Daily Login Failure
+        * AWS - Login Failure From Unusual Country Due To Multi Factor Authentication
+    * Google Workspace
+        * Google Workspace - User Change/Update
+        * Google Workspace - Enterprise Group Change/Update
+        * Google Workspace - Enterprise Group Membership Change/Update
+        * Google Workspace - Role Change/Update
+        * Google Workspace - Successful Login From Unusual Country
+        * Google Workspace - Suspicious Login Activity by User
+        * Google Workspace - Daily Login Failure
+        * Google Workspace - Alerts Center Alert
+        * Google Workspace - Google Drive objects shared Outside or with External User
+        * Google Workspace - Suspicious File Shared by External User on Google Drive
+    * Email:
+        * Email - Suspicious Subject or Attachment
+        * Email - With Known Abuse Web Service Link
+    * Sophos:
+        * Sophos - Failed to CleanUp Potentially Unwanted Application by Sophos
+    * Ransomware
+        * Ransomware - Endpoint Compromise - Malicious Package Found
 
 
 * ### Enhancements  
 
-    * #### Device Inventory
-        * Enhanced the device inventory logic by: 
-            * Updating the device matching and hostname matching logic.
-            * Modifying its data structure.
-            * Adding the functionality to auto merge devices.
-            * Updating the functionality to manual merge devices.
-        * The "Asset Intelligence" dashboard has been renamed to "Intelligence".
-        * Added the "Device Inventory - Splunk Internal" saved search to collect the forwarder information available in splunk.
-        * Added the "Device Inventory Merge Similar Devices" saved search to auto merge the similar devices.
-
-    * Improved the row expansion table design by removing empty rows and adding borders to the table.
-
-    * Reduced the severity of the "Authentication - Excessive Failed VPN Logins for a User" alert for unknown user.
-
-    * The "O365 - Azure Active Directory - Group Change/Update" and "O365 - Azure Active Directory - GroupMembership Change/Update" alerts have been updated to capture the appropriate events.
-
-    * Updated the severity of "O365 - Login Failure From Unusual Country Due To Multi Factor Authentication" and "O365 - Login Failure Due To Multi Factor Authentication" alerts.
+    * The "Gsuite" dashboard has been renamed to "Google Worspace".
```

Comment:
Add this as `###` points. -> `GSuite has been replaced with Google Workspace`.
* GSuite Add-on is no longer supported, kindly remove related Add-ons.
* Install the Google Workspace Add-on and configure it for data collection (link to data onboarding section).
* Removed below alerts and replaced with related alerts.
* Removed GSuite dashboard and replaced with Google Workspace dashboard.